### PR TITLE
Add Windows and Mobile Details popup

### PR DESCRIPTION
**Pull request by benardop**
## Changes I made in this PR
-  Add details pop-up window for desktop website
-  Add details pop-up for the mobile website

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -820,6 +863,191 @@ nav .closebtn {
...
+  overflow: auto;
+  left: 50%;
+  top: 1rem;
+  transform: translate(-50%, 0);
+  width: 90%;
+  height: 95vh;
+  border-radius: 16px;
+  padding: 25px;
+  background-color: #fff;
+}
```
Comment (mwanawabangona) [inline css/styles.css:916]: - [ ] `required`
- [ ] Good job so far 👏 👏 
- [ ] However i would suggest you make changes in accordance with the `figma template` 
- [ ] Please make sure `all` the popups follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., `font`, `colours`, `images`, `text`, `margins`) see screenshot for comparisons

`figma template`
figma pop-up

`your desktop popup`
deaktop template

File: css/styles.css [hunk 2]
```diff
@@ -820,6 +863,191 @@ nav .closebtn {
...
+  position: fixed;
+  overflow: auto;
+  left: 50%;
+  top: 1rem;
+  transform: translate(-50%, 0);
+  width: 90%;
+  height: 95vh;
+  border-radius: 16px;
+  padding: 25px;
+  background-color: #fff;
```
Comment (mwanawabangona) [inline css/styles.css:915]: - [ ] `required`
- [ ] as already mentioned above please make adjustment according to the figma template
- [ ] for all the `mobile pop-up cards`
- [ ]  Please make sure `all` the popups follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., `font`, `colours`, `images`, `text`, `margins`) see screenshot for comparisons

`your template`
mobile template

`figma template`
mobile pf

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @benardop ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Comparing both designs the `figma template` and  your template and going through the requirements step by step.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 2 above)
Comment (benardop) [reply css/styles.css:915]: @mwanawabangona Could you please check once again the popup I have made in the screenshot you have taken above? If you scroll down, you will the screen coming exactly as it is with all the buttons indicated. Kindly very that.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @benardop , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- Ensure your card image is showing and remove the extra space surrounding the image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**